Mathlib/Analysis/SpecialFunctions/Pow/Continuity.lean

```diff
@@ -249,6 +249,9 @@ theorem continuousAt_rpow_const (x : ℝ) (q : ℝ) (h : x ≠ 0 ∨ 0 < q) :
   · exact (continuous_id'.prod_mk continuous_const).continuousAt
 #align real.continuous_at_rpow_const Real.continuousAt_rpow_const
 
+@[fun_prop]
+theorem continuous_rpow_const {q : ℝ} (h : 0 < q) : Continuous (fun x : ℝ => x ^ q) :=
```

Comment:
Could you assume that `q` is nonnegative instead? (Should be trivial for `q = 0`).

Reply:
Thanks! I also generalized the previous lemma.